### PR TITLE
What is time travel

### PR DESCRIPTION
**Pull request by jasonLaster**
- cleans up the intro
- adds a new recording overhead section, replay protocol, replay performance, and replay analsysis section

This is still a draft, but i think it does a good job of documenting many of the things we talk about when folks are interested in understanding the magic / mystery beind time travel.

### CODE REVIEW

File: vercel.json
Comment (jonbell-lot23) [inline vercel.json:1]: We have a redirect problem here -- check line 8, where we are redirecting "what is" to "why":

```
{
      "source": "/time-travel-intro/what-is-time-travel",
      "destination": "/time-travel-intro/why-time-travel"
    },
```

So we gotta drop that bit @jasonLaster

Comment (jonbell-lot23) [reply vercel.json:1]: Fixed

**Review by jonbell-lot23 — APPROVED** [review]
Looks good! I found the examples at the top to be really helpful for me, and I think there's a good amount of detail throughout.

The naive versus forking image could be improved but oh gosh do I have other things that are higher priority, so we'll let it be for now.

I committed a bunch of little typo, comma, weasel-word additions to tighten some things up.

**Comment by crutchcorn** [comment]
This reads well and I think is overall an improvement, but I have some broader thoughts about the structure of our sidebar and the contents within that I think these page updates raise even further for me. Opening a new issue to discuss further

**Pull request merged**